shoop/front/apps/simple_search/views.py

```diff
@@ -6,14 +6,14 @@
 # This source code is licensed under the AGPLv3 license found in the
 # LICENSE file in the root directory of this source tree.
 from django import forms
+from django.core.cache import cache
 from django.db.models import Q
 from django.utils.encoding import force_bytes
 from django.views.generic import ListView
-import hashlib
-from shoop.core.models.products import Product
-from django.core.cache import cache
-from shoop.front.utils.views import cache_product_things
+from shoop.core.models import Product
 from shoop.front.utils.product_sorting import PRODUCT_SORT_CHOICES, sort_products
+from shoop.front.utils.views import cache_product_things
+import hashlib
```

Comment:
While fixing import statement ordering make it conform PEP-8 groping too, i.e. first stdlib imports, then third party imports, then local imports (= shoop imports in our case), each group in alphabetical order and groups separated by a blank line.